core/engine_test.go

```diff
@@ -652,6 +653,19 @@ func TestRunTags(t *testing.T) {
 		return "the rainbow"
 	}
 
+	// Act
+	errC := make(chan error)
+	go func() { errC <- run() }()
+
+	// Assert
+	select {
+	case <-time.After(10 * time.Second):
```

Comment:
I think we could decrease this timeout, probably for getting a feedback faster in case we would hit the case

Reply:
why? the timeout is here just in case, so we don't hit the global 120s one :man_shrugging: